src/providers/events.js

```diff
@@ -75,7 +82,10 @@ export function createEventsObject(openmct, parentKey, namespace) {
                 {
                     key: 'message',
                     name: 'Message',
-                    format: 'string'
+                    format: 'string',
+                    hints: {
```

Comment:
Can you write some documentation on what this hint is and how it is used?
`hints: {
      label: 0
  }`